src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,152 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    public static void main(String[] args) {
+        String line;
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("____________________________________________________________");
+        System.out.println("    Hello! I'm Duke");
+        System.out.println("    What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        SaveHandler saveHandler = new SaveHandler();
+        ArrayList<Task> list = saveHandler.load();
+
+        boolean isExitCommandReceived = false;
+
+        while (!isExitCommandReceived) {
+            line = in.nextLine();
+            try  {
+                Command command = new Command(line);
+
+                if (command.getCommandType() == CommandType.CREATE_TODO) {
```

Comment:
Consider refactoring the if-else structure out. Also, could using switch-case for this scenario be more suitable and readable?